docs/scripts/buildApi.ts

```diff
@@ -397,18 +397,21 @@ async function buildDocs(options: {
     `import React from 'react';
 import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
 import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
+import markdownRaw from './${kebabCase(reactAPI.name)}.md';
 
-const pageFilename = 'api/${kebabCase(reactAPI.name)}';
-const requireRaw = require.context('!raw-loader!./', false, /\\/${kebabCase(reactAPI.name)}\\.md$/);
+const pageFilename = '${kebabCase(reactAPI.name)}';
```

Comment:
why not include api? Note that it's used in https://github.com/mui-org/material-ui/blob/7db816a0540d6286f6d69f8d52b34602d2491e41/docs/src/modules/components/EditPage.js#L14. Maybe we could leverage API to point to a different file?

Reply:
> Note that it's used in

1. It isn't used there - we link to the component path (and on the wrong branch name `staging`)
2. Even if it was used there instead of the component path, the path is incorrect (should be `api-docs`)
3. Even if it was used there, and was the correct path, there are no API docs to translate 😄